pl-csb-model/config.ini

```diff
@@ -1,9 +1,13 @@
 [MODEL]
-PretrainedModelName = facebook/nllb-200-distilled-600M
-OutputModelName = pl-csb-model-v1
+pretrained_model_name = facebook/nllb-200-distilled-600M
+output_model_name = pl-csb-model-v1
 
 [TRAINING]
-BatchSize = 8
-MaxLength = 128
-WarmupSteps = 100
-TrainingSteps = 1000
+batch_size = 8
+max_length = 128
+warmup_steps = 100
+training_steps = 100
+
+[DATA]
+training_data_file = ../pl-csb-data/data/output/train.tsv
```

Comment:
Let's follow a similar approach as in https://github.com/kashubian-translator/pl-csb-data/blob/master/pl-csb-data/config.ini:
```python
[DIRECTORIES]
output_data_dir = ../pl-csb-data/data/output/

[MODEL]
pretrained_model_name = facebook/nllb-200-distilled-600M
output_model_name = pl-csb-model-v1

[TRAINING]
batch_size = 8
max_length = 128
warmup_steps = 100
training_steps = 100

[DATA]
training_data_file = ${DIRECTORIES:output_data_dir}/train.tsv
Comment
evaluation_data_file = ${DIRECTORIES:output_data_dir}/eval.tsv
```
Please remember to update the config loader as well https://github.com/kashubian-translator/pl-csb-data/blob/master/pl-csb-data/config.ini